Blackfynn/data-portal
# Fix mobile featured dataset #1c44ph

# Description

The purpose of this PR is to fix the featured dataset carousel on the homepage on mobile.

## Type of change


# How Has This Been Tested?

- Go to the homepage
- Scroll to the bottom of the page to see the featured datasets carousel
- Should be normal
- Reduce browser size to `< 768px`
- Featured datasets carousel should be responsive with the image above the copy.


# Checklist:
